.github/workflows/sensei.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       CC: clang
       CXXFLAGS: "-Werror -Wshadow -Woverloaded-virtual -Wunreachable-code -fno-operator-names"
       CMAKE_GENERATOR: Ninja
-      CMAKE_PREFIX_PATH: /root/install/sensei/develop/lib/cmake
+      CMAKE_PREFIX_PATH: /root/install/sensei/v4.0.0/lib/cmake
```

Comment:
My apologies, it is `lib64` not `lib` now for some reason.

Reply:
no worries, I knew about that, it's because we've adopted the GNUInstallDirs conventions from CMake.